ui/core/components/individual_sim_ui/gem_summary.ts

```diff
@@ -26,6 +27,20 @@ export class GemSummary extends Component {
 			header: { title: 'Gem Summary' },
 		});
 		player.gearChangeEmitter.on(() => this.updateTable());
+
+		const headerElement = this.container.headerElement;
+		if (headerElement) {
+			const unequipButton = document.createElement('button');
```

Comment:
I was hesitated to do so because making one component into tsx is inconsitent to the other files if no plan to make all of them the tsx. But that is my opinion.

Reply:
You have to start somewhere :D That's how I did it for the Cata repo as well. If you never do it you'll also never change it

Reply:
Sounds good, I will proceed